ValidationLibrary.Rules/HasCodeownersRule.cs

```diff
@@ -62,6 +62,15 @@ public async Task<ValidationResult> IsValid(IGitHubClient client, Repository rep
             return new ValidationResult(RuleName, "Add CODEOWNERS file & add at least one owner.", !string.IsNullOrWhiteSpace(codeownersContent.Content), DoNothing);
         }
 
+        public Dictionary<string, string> GetConfiguration()
+        {
+            return new Dictionary<string, string>
+            {
+                 { "PullRequestTitle", RuleName },
```

Comment:
This rule doesn't create a Pull Request, so `PullRequestTitle` doesn't make sense here